test/e2e/preset_test.go

```diff
@@ -300,11 +298,12 @@ func createConfigForWorkspace(workspaceObj *kaitov1beta1.Workspace) {
 		return
 	}
 
-	handler := sku.GetCloudSKUHandler(consts.AzureCloudName)
-	gpuConfig := handler.GetGPUConfigBySKU(workspaceObj.Resource.InstanceType)
-	if gpuConfig == nil || (gpuConfig.GPUCount <= 1 && lo.FromPtr(workspaceObj.Resource.Count) <= 1) {
-		return
-	}
+	// TODO: uncomment the following lines when A10 GPU support is added
+	// handler := sku.GetCloudSKUHandler(consts.AzureCloudName)
+	// gpuConfig := handler.GetGPUConfigBySKU(workspaceObj.Resource.InstanceType)
+	// if gpuConfig == nil || (gpuConfig.GPUCount <= 1 && lo.FromPtr(workspaceObj.Resource.Count) <= 1) {
+	// 	return
+	// }
```

Comment:
Can we uncomment this now because you added A10 support?

Reply:
The probing logic has some compatibility issue this this hardware.
Let's workaround this issue at first to unblock e2e test.
I will fix that issue later.